std/php7/NativeArray.hx

```diff
@@ -21,6 +21,67 @@
  */
 package php7;
 
-extern class NativeArray implements ArrayAccess<Dynamic> {
+import haxe.extern.EitherType;
 
-}
\ No newline at end of file
+using php7.Global;
+
+
+/**
+	Native PHP array.
+**/
+@:coreType @:runtimeValue abstract NativeArray {
+	public inline function new()
+		this = untyped __php__("[]");
+
+	@:arrayAccess function get(key:Scalar):Dynamic;
+	@:arrayAccess function set(key:Scalar, val:Dynamic):Dynamic;
+
+	public inline function keys():NativeIndexedArray<EitherType<String,Int>>
```

Comment:
Unless I overlook something where they are really needed, I'd say yes.
They are just duplicated and confusing this way.
Oh wait: There are some typed versions with`<T>` in NativeIndexedArray, that might be useful, but the type might be inferred in the Array function also, have to try.